client/my-sites/activity/activity-log-banner/upgrade-banner.jsx

```diff
@@ -26,10 +27,12 @@ class UpgradeBanner extends Component {
 						feature={ WPCOM_FEATURES_FULL_ACTIVITY_LOG }
 						href={ `/checkout/${ siteSlug }/${ PRODUCT_UPSELLS_BY_FEATURE[ FEATURE_ACTIVITY_LOG ] }` }
 						title={ translate( 'Unlock more activities now' ) }
-						description={ translate(
-							'With your free plan, you can monitor the 20 most ' +
-								'recent events on your site. Upgrade to a paid plan to ' +
-								'unlock powerful features:'
+						description={ preventWidows(
+							translate(
+								'You currently have access to the 20 most recent events ' +
+									'on your site. Upgrade to Jetpack VaultPress Backup ' +
+									'or Jetpack Security to unlock powerful features:'
```

Comment:
I don't quite understand why these strings are split up and then added back together 🤔 Same with the one below this one

Reply:
The only reason, I believe, is to keep the line length in the editor from being suuuper long and having to side scroll way to the right to read it. (But without having actual line breaks in the string)